### PR TITLE
Add documentation to deploy Play! on Scalingo PaaS

### PR DESCRIPTION
**Pull request by Soulou**
# Pull Request Checklist
- [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
- [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
- [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
- [x] Have you [squashed your commits]? (Optional, but makes merge messages nicer.)
- [x] Have you added copyright headers to new files?
- [n/a] Have you checked that both Scala and Java APIs are updated?
- [n/a] Have you updated the documentation for both Scala and Java sections?
- [n/a] Have you added tests for any changed functionality?
## Purpose

This pull request aims at guiding users to deploy Play! application on the Platform as a Service [Scalingo](https://scalingo.com)

### CODE REVIEW

File: documentation/manual/working/commonGuide/production/cloud/Deploying-Scalingo.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+<!--- Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com> -->
+# Deploying to Scalingo
+
+[Scalingo](https://scalingo.com/) is a Platform as a Service hosting solution –
+it is technology agnostic, your can deploy Scala, Java as well as Ruby or
+Node.JS apps.
+
+The platform supports Play! apps natively, you have nothing to add to your project and
```
Comment (gmethvin) [inline documentation/manual/working/commonGuide/production/cloud/Deploying-Scalingo.md]: I'm not sure I understand this sentence. It probably would be enough to say:
> The platform supports Play apps natively. No additional configuration is required on your project. The setup process is explained here.

File: documentation/manual/working/commonGuide/production/cloud/Deploying-Scalingo.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+here is to setup your app on the platform.
+
+In this guide, all the operations can be done through Scalingo web dashboard or with its
+command line tool (it's to your liking).
```
Comment (gmethvin) [inline documentation/manual/working/commonGuide/production/cloud/Deploying-Scalingo.md]: Should the parenthetical say "(*if* it's to your liking)"?

File: documentation/manual/working/commonGuide/production/cloud/Deploying-Scalingo.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+```text
+db.default.url=${?JDBC_DATABASE_URL}
+db.default.username=${?JDBC_DATABASE_USERNAME}
+db.default.password=${?JDBC_DATABASE_PASSWORD}
+```
+
+Note that the creation of a Procfile is not actually required by Scalingo, as
+Scalingo will look in your Play application's conf directory for an
+`application.conf` file in order to determine that it is a Play application.
```
Comment (gmethvin) [inline documentation/manual/working/commonGuide/production/cloud/Deploying-Scalingo.md]: This seems like a poor assumption to make. `application.conf` can be present in any almost app that uses Typesafe Config, and may not be present if you've configured a custom configuration file.

It's quite common that you want to [specify an alternate configuration file](https://www.playframework.com/documentation/2.5.x/ProductionConfiguration#Specifying-an-alternate-configuration-file) in production. That is often where production-specific database configuration stuff like this is placed, but it also may be used to configure other production settings. Am I correct that the Procfile is required if you want to do that? If so, we should mention it in a separate section.

**Comment by wsargent** [comment]
Closed due to lack of activity

**Pull request closed without merging**